values/encoding/compression_bench_test.go

```diff
@@ -10,22 +10,23 @@ import (
 )
 
 const (
-	testDataFilePath       = "./testdata/testdata.json"
-	compressedDataFilePath = "../decoding/testdata/testdata.json.compressed"
+	testDataFilePath = "./testdata/testdata.json"
 )
 
 func compressData(data []byte) error {
-	out, err := os.OpenFile(compressedDataFilePath, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0644)
+	out, err := ioutil.TempFile("/tmp", "*")
```

Comment:
nit: Just pass in an empty string as the first arg since `/tmp` may not exist on some machines, the pattern perhaps is better to use `*.decompressed`?